docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.horizontal-card.scss

```diff
@@ -0,0 +1,147 @@
+.horizontal-card {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 2.75rem;
+
+  @include grid-media-min("md") {
+    flex-direction: row;
+  }
+
+  @include grid-media-min("lg") {
+    align-items: center;
+  }
+
+  &__category {
+    @include hb_subheading;
+    text-transform: uppercase;
+    margin-bottom: 0.3125rem;
+
+    ul {
```

Comment:
this ul and li are only output for categories if this is configured carefully in the admin side